scheduler/manifests/install/base/deployment.yaml

```
@@ -37,17 +37,32 @@ spec:
               port: 10259
               scheme: HTTPS
           resources:
+            limits:
+              cpu: '300m'
+              memory: '40Mi'
```

Just to be safe when we have to deal with huge applications
```suggestion
              memory: '100Mi'
```